plugins/CMakeLists.txt

```diff
@@ -75,8 +76,8 @@ IF("${PLUGIN_LIST}" STREQUAL "")
 		stereo_enhancer
 		stereo_matrix
 		stk
-		vestige
 		vst_base
+		vestige
```

Comment:
I think there's no reason to move this.

Reply:
There is, `vestige` links to `vstbase`, so `vst_base` needs to be processed first. Otherwise, the target doesn't exist when `vestige` tries to link to it.